Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/submissions/use-submissions-details.tsx

```diff
@@ -209,12 +165,14 @@ const SubmissionsDetailsProvider = ({ children }: ISubmissionsDetailsProviderPro
                 return;
             }
 
-            const { validationResult: newValidationResult } = apiSubmissionDetails;
+            if (!isNil(apiSubmissionDetailsError)) {
+                setSubmissionDetailsError(apiSubmissionDetails as unknown as IErrorDataType);
```

Comment:
Same as above

Reply:
I need to cast it to unknown first to be able to cast it to IErrorDataType.